clusters/build-clusters/common/assets/pod-scaler-admission.yaml

```diff
@@ -0,0 +1,120 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: "pod-scaling.ci.openshift.io"
+webhooks:
+- name: "pod-scaling.ci.openshift.io"
```

Comment:
yeah we should set objectSelector or namespaceSelector before we deploy this

Reply:
I'm going to do it manually today on one cluster only